### PR TITLE
Adding eclipse shortcut key config

### PR DESCRIPTION
**Pull request by senisioi**
This set of shortcut keys is largely based on the existing ones for eclipse from [here](http://eclipse-tools.sourceforge.net/Keyboard_shortcuts_(3.0).pdf) and [here](https://shortcutworld.com/en/Eclipse/win/all)

### CODE REVIEW

File: cfg/eclipse-keys.lua [hunk 1]
```diff
@@ -0,0 +1,100 @@
+-- In your user / system configuration file simply type: include "eclipse-keys.lua"
```
Comment (madmaxoft) [inline cfg/eclipse-keys.lua]: Does `include` really work? I'd expect `require` instead.

Comment (senisioi) [inline cfg/eclipse-keys.lua]: Seems to work, based on [this](https://studio.zerobrane.com/doc-configuration#configuration-file-commands) document

Comment (madmaxoft) [inline cfg/eclipse-keys.lua]: Okay, my bad, sorry.

Comment (pkulchenko) [inline cfg/eclipse-keys.lua]: Yes, `include` should be used as it does things that `require` doesn't do in this context: searches locally and relatively to the config where it's called plus in one places known to store config, provides the expected environment, and few other things.

**Comment by pkulchenko** [comment]
Merged; thank you. I also updated instructions and simplified the code a bit (post-merge).

**Pull request merged**